test/parallel/test-uv-errno.js

```diff
@@ -14,7 +14,7 @@ const keys = Object.keys(uv);
 
 assert.strictEqual(uv.errname(-111111), 'Unknown system error -111111');
 
-keys.forEach((key) => {
+for (const key of keys) {
   if (!key.startsWith('UV_'))
     return;
```

Comment:
The `return` statement makes sense with `forEach`, but not with `for ... of`. Please use `continue` instead.